rotkehlchen/tests/utils/factories.py

```diff
@@ -38,6 +40,9 @@
 
 DEFAULT_START_TS = Timestamp(1451606400)
 ZERO_TIMESTAMP_MS = TimestampMS(0)
+ADDRESS_ETH = string_to_evm_address('0x9D904063e7e120302a13C6820561940538a2Ad57')
+ADDRESS_MULTICHAIN = string_to_evm_address('0x368B9ad9B6AAaeFCE33b8c21781cfF375e09be67')
+ADRESS_ROSE = string_to_evm_address('0x3D61AEBB1238062a21BE5CC79df308f030BF0c1B')
```

Comment:
Why do you name this `ADDRESS_ROSE` when in api/test_addressbook.py:611 there a different one with that name?